app/controllers/drinks_controller.rb

```diff
@@ -9,9 +9,23 @@ def index
     end
 
     def create
-        Drinks.create(name:params[:name])
+        #current_user to use when not in testing anymore
+        drink = Drink.find_by(api_id:params[:api_id])
+        drink = Drink.create(drinks_params) if drink.nil? 
+        favorite = FavoriteDrink.create(user: params[:user_id], drink: drink) 
```

Comment:
```suggestion
        favorite = FavoriteDrink.create(user: current_user.id, drink: drink) 
```
The devise already allows you to access the current_user so it doesn't need to be provided by params

Reply:
Hey, Mina I was just doing this for testing, when we actually hook the front-end to backend ill switch it over to current_user.